core/storage/email/gae_models.py

```diff
@@ -18,6 +18,7 @@
 
 import datetime
 
+from core.domain import feedback_domain
```

Comment:
I thought so too! Actually it is used to create a suggestion (suggestion has an exploration_id field). Shall I revert the changes to pass an exploration_id separately instead? 

Also this function will be deleted after the new suggestions framework comes in place and then the code should be properly layered.

Reply:
Good intuition! Yes, if you can pass the exploration_id separately, that would be better.

Reply:
Actually, wait a sec. That's the case for the feedback models, but I'm not sure I understand how it applies here. Wouldn't you need some kind of migration for this, since the thread ID is part of the model instance ID?

Reply:
Oh my bad. Wrong location for the message. The initial ID for the model was user_id.exp_id.thread_id (partial thread_id). Now I have made it user_id.thread_id. (full thread_id)

So the ID is the same essentially. Only when we generalise threads, we will need a migration. 

But here I think we will have to revert to having the .split() itself and track it. Not sure of any better approaches.

Reply:
Yep, for this one, I agree with you.

Reply:
Reverted.

Reply:
Also made a note of this refactor in #4995